source/jormungandr/jormungandr/tests/utils_test.py

```diff
@@ -63,3 +66,108 @@ def get(self, url, *args, **kwargs):
             url += "?{}".format(urlencode(kwargs.get('params'), doseq=True))
 
         return MockResponse(self.responses[url][0], self.responses[url][1], url)
+
+
+class FakeUser:
+    """
+    We create a user independent from a database
+    """
+    def __init__(self, name, id, have_access_to_free_instances=True, is_super_user=False, is_blocked=False, shape=None):
+        """
+        We just need a fake user, we don't really care about its identity
+        """
+        self.id = id
+        self.login = name
+        self.have_access_to_free_instances = have_access_to_free_instances
+        self.is_super_user = is_super_user
+        self.end_point_id = None
+        self._is_blocked = is_blocked
+        self.shape = shape
+
+    @classmethod
+    def get_from_token(cls, token):
+        """
+        Create an empty user
+        """
+        return user_in_db[token]
+
+    def has_access(self, instance_name, api_name):
+        """
+        This is made to avoid using of database
+        """
+        return authorizations[self.login][instance_name][api_name]
+
+    def is_blocked(self, datetime_utc):
+        """
+        Return True if user is blocked else False
+        """
+        return self._is_blocked
+
+def geojson():
+    return '{"type": "Feature", "geometry": ' \
+           '{"type": "Point", "coordinates": [102.0, 0.5]}, "properties": {"prop0": "value0"}}'
+
+user_in_db = {
```

Comment:
in your bragi test you only need 2 users no ?

I'm not a fan of putting test specific stuff in utils, I think it makes the tests less readable